### DISCUSSION TITLE
Allow providing path to CAcert file, instead of actual content

### DISCUSSION THREAD

**Discussion by ctron** [grafana/grafana#44272] [Data Sources]
It is currently possible to provide the content of a CAcert file for an HTTP based datasource.

However, if you have this file inside of a cluster, generated by some other process, copying that content over to the datasource YAML for provisioning doesn't make much sense, and can be a tedious task.

Besides, e.g. Postgres has the ability to configure the path to a file which contains the content. So it looks like other datasources do support his already. HTTP based datasources (like prometheous) should too.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
